pubspec.yaml

```diff
@@ -269,6 +275,7 @@ dependency_overrides:
     git:
       url: https://github.com/linagora/matrix_link_text.git
       ref: twake-supported
+  web: ^0.5.0
```

Comment:
we use it for what?

Reply:
Nevermind, I forgot to remove it but it was for testing purpose of flutter_web_auth_2 alphas which uses this package.